src/crimson/common/tri_mutex.cc

```diff
@@ -72,7 +63,7 @@ tri_mutex::~tri_mutex()
 seastar::future<> tri_mutex::lock_for_read()
 {
   if (try_lock_for_read()) {
-    return seastar::make_ready_future<>();
+    return seastar::now();
```

Comment:
this change does not belong to this commit.

Reply:
splitted